openupgradelib/openupgrade.py

```diff
@@ -499,6 +500,44 @@ def copy_fields_multilang(cr, destination_model, destination_table,
         )
 
 
+def remove_tables_fks(cr, *tables):
+    """Remove foreign keys declared in ``tables``.
+
+    This is useful when a table is not going to be used anymore, but you still
+    don't want to delete it.
+
+    If you keep FKs in that table, it will still get modifications when other
+    tables are modified too; but if you're keeping that table as a log, that
+    is a problem. Also, if some of the FK has no index, it could slow down
+    deletion in other tables, even when this one has no more use.
+
+    .. HINT::
+        This method removes FKs that are *declared* in ``tables``,
+        **not** FKs that *point* to those tables.
+
+    :param *str tables:
```

Comment:
This star is taken as emphasis start causing /home/travis/build/OCA/openupgradelib/openupgradelib/openupgrade.py:docstring of openupgradelib.openupgrade.remove_tables_fks:15:Inline emphasis start-string without end-string.

Reply:
Fixed, thanks.